ClimsoftVer4/ClimsoftVer4/formMetadata.vb

```diff
@@ -6,70 +6,71 @@
     Dim sql As String
     Dim rec As Integer
     Dim Kount As Integer
+    Dim ActiveTab As Integer
     'Dim maxRows As Integer
 
     Private Sub CloseToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles CloseToolStripMenuItem.Click
         Me.Close()
     End Sub
 
-    Private Sub GroupBox2_Enter(sender As Object, e As EventArgs) Handles grpStation.Enter
 
-    End Sub
 
     Private Sub formMetadata_Click(sender As Object, e As EventArgs) Handles Me.Click
-        MsgBox(TabMetadata.SelectedIndex)
+        'MsgBox(TabMetadata.SelectedIndex)
     End Sub
 
     Private Sub formMetadata_Load(sender As Object, e As EventArgs) Handles MyBase.Load
         dbConnectionString = frmLogin.txtusrpwd.Text
         dbconn.ConnectionString = dbConnectionString
         dbconn.Open()
 
-        'Sql = "SELECT * FROM station"
-        'da = New MySql.Data.MySqlClient.MySqlDataAdapter(sql, dbconn)
-        'da.Fill(ds, "station")
-
-        'MsgBox(ds.Tables("station").Rows.Count)
-
+        SetDataSet("station")
+        rec = 0
+        populateStations("station", rec, Kount)
+        populateSearchStation()
+        populateSearchElement()
     End Sub
 
     Sub SetDataSet(tbl As String)
         Dim sql As String
         Try
             sql = "SELECT * FROM " & tbl
             da = New MySql.Data.MySqlClient.MySqlDataAdapter(sql, dbconn)
+            ds.Clear()
             da.Fill(ds, tbl)
             Kount = ds.Tables(tbl).Rows.Count
         Catch ex As Exception
-            MsgBox(ex.Message)
+            'MsgBox(ex.Message)
         End Try
 
     End Sub
 
     Private Sub TabStation_Click(sender As Object, e As EventArgs) Handles TabStation.Click
-        MsgBox("Station")
```

Comment:
Second change - comments the message box when user clicks within the Station metadata tab